pandas/io/html.py

```diff
@@ -554,8 +553,7 @@ def _parse_td(self, row):
         return row.xpath('.//td|.//th')
 
     def _parse_tr(self, table):
-        expr = './/tr[normalize-space()]'
```

Comment:
It wasn't clear to me why `normalize-space()` was added here. It is inconsistent with how bs4 parses `tr` elements and was actually causing a failure in `test_computer_sales_page`.

Should the need to strip trailing / leading whitespace come back up I think it would be better done in the base class than only implementing here in lxml